src/components/serviceUI/styledP.jsx

```diff
@@ -0,0 +1,9 @@
+import styled from 'styled-components';
+
+const StyledP = styled.p`
+	color: #f9f6ee;
```

Comment:
If you later use more colors later on, you could think about using CSS variables.

Reply:
Hey Lene, i will do the styling in the 3 week of capstone, all these colors are "placeholders"